### PR TITLE
Add docs for tracking PRs assigned for review through the Zulip chat

### PR DESCRIPTION
**Pull request by apiraino**
We are implementing a mechanism to track and manage pull requests assigned for review to Rust contributors.

For now, the only interface to query your work queue is the Zulip chat.

This patch starts documenting how to interact with the Zulip chat bot.

### CODE REVIEW

**Comment by apiraino** [comment]
cc: @jackh726 that is helping me with the triagebot work implementing the PR assignment

File: src/triagebot/pr-assignment-tracking.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Configuration
+
+Tracking the PR assignment is enabled on the git repository by having a `[pr-tracking]` table in `triagebot.toml`. No additional configuration is needed.
+
+## Usage
+
+Open a Direct Message session with the `triagebot` and send a message with one of these commands:
+
+* `work` --- Will emit an error and show the available commands
+* `work show` --- Will show your Github username and a list of pull requests assigned to you for review (on the `rust-lang/rust` git repository)
```
Comment (ehuss) [inline src/triagebot/pr-assignment-tracking.md:20]: Does it have to be rust-lang/rust, or can it be an repo that has it configured?

Comment (apiraino) [reply src/triagebot/pr-assignment-tracking.md:20]: We are not yet ready to manage more than one git repository at a time with this feature (so we are going to test `rust-lang/rust` first)

**Comment by jackh726** [comment]
This is probably fine, but I think we should discourage people from using these commands (especially upcoming work) for now - at least until we test a bit.

**Review by jackh726 — APPROVED** [review]

**Pull request merged**